src/ioctls/system.rs

```diff
@@ -97,11 +97,27 @@ impl Kvm {
     /// AArch64 specific call to get the host Intermediate Physical Address space limit.
     ///
     /// Returns 0 if the capability is not available and an integer >= 32 otherwise.
-    #[cfg(any(target_arch = "aarch64"))]
+    #[cfg(target_arch = "aarch64")]
     pub fn get_host_ipa_limit(&self) -> i32 {
         self.check_extension_int(Cap::ArmVmIPASize)
     }
 
+    /// AArch64 specific call to get the number of supported hardware breakpoints.
+    ///
+    /// Returns 0 if the capability is not available and a positive integer otherwise.
```

Comment:
Does the integer bear any meaning? Can we return a bool in case it doesn't have any useful information?

Reply:
The integer is the number of breakpoints.

Reply:
The integer is useful in supporting the debug of guest kernel. 
On x86, the max number of breakpoints is 4, while on arm the number is not fixed but is indicated by this API.